formal/promela/models/threadq/srccpy/mrsp-scenarios/mrsp-Init.c

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
Do we really need the "srccpy" directory?

Reply:
No, we don't.